engine/src/test/java/org/camunda/bpm/engine/test/api/task/TaskIdentityLinksTest.java

```diff
@@ -253,4 +265,61 @@ public void testDeleteOwner() {
     taskService.deleteTask(task.getId(), true);
   }
 
+  @Test
+  public void testAssigneeGetIdentityLinksInCompleteListener() {
+    // given
+    BpmnModelInstance model = Bpmn.createExecutableProcess("process")
+                                          .startEvent()
```

Comment:
Formatting is a bit odd here and in the second test method, too.

Reply:
@danielkelemen, you mean the model elements?

Reply:
yes, it's indented quite a bit :)